README.md

```diff
@@ -1,2530 +1,30 @@
-# APACHE NUTTX
+<h1 align="center">
+<img src="https://raw.githubusercontent.com/apache/nuttx/master/Documentation/_static/NuttX320.png" width="150">
+</h1><br>
 
-* Introduction
-* Community
-  - Getting Help
-  - Mailing Lists
-  - Issue Tracker
-  - Source Code
-  - Website Source Code
-* Environments
-  - Installing Cygwin
-  - Ubuntu Bash under Windows 10
-  - Using macOS
-* Installation
-  - Download and Unpack
-  - Semi-Optional apps/ Package
-  - Installation Directories with Spaces in the Path
-  - Downloading from Repositories
-  - Related Repositories
-  - Notes about Header Files
-* Configuring NuttX
-  - Instantiating "Canned" Configurations
-  - Refreshing Configurations
-  - NuttX Configuration Tool
-  - Finding Selections in the Configuration Menus
-  - Reveal Hidden Configuration Options
-  - Make Sure that You are on the Right Platform
-  - Comparing Two Configurations
-  - Making defconfig Files
-  - Incompatibilities with Older Configurations
-  - NuttX Configuration Tool under DOS
-* Toolchains
-  - Cross-Development Toolchains
-  - NuttX Buildroot Toolchain
-* Shells
-* Building NuttX
-  - Building
-  - Re-building
-  - Build Targets and Options
-  - Native Windows Build
-  - Installing GNUWin32
-* Cygwin Build Problems
-  - Strange Path Problems
-  - Window Native Toolchain Issues
-* Documentation
+![POSIX Badge](https://img.shields.io/badge/POSIX-Compliant-green?style=flat&label=POSIX)
+![Issues Tracking Badge](https://img.shields.io/badge/issue_track-github-blue?style=flat&label=Issue%20Tracking)
+[![GitHub Build Badge](https://github.com/apache/nuttx/workflows/Build/badge.svg)](https://github.com/apache/nuttx/actions/workflows/build.yml)
+[![GitHub Documentation Build Badge](https://github.com/apache/nuttx/workflows/Build%20Documentation/badge.svg)](https://github.com/apache/nuttx/actions/workflows/doc.yml)
```

Comment:
Other badge ideas: licence, link to release, link to doc

Reply:
Do we need `Documentation Build Badge` ?

Reply:
I really can't see why we would need it, except for another badge to make it look "modern"

Reply:
I changed the "Build Documentation" badge to open the actual documentation.
Also added license. What would link to release be?

Reply:
there is `[![GitHub release](https://img.shields.io/github/release/apache/nuttx.svg)](https://github.com/apache/nuttx/releases)` but it doesn't seem to work with NuttX. Probably this works only with Github releases which is empty in our case https://github.com/apache/nuttx/releases